File: WEB/exam sample/java/catalog-core/src/main/java/catalog/core/model/SoftwareDeveloper.java
package catalog.core.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Entity;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class SoftwareDeveloper extends BaseEntity<Long> {
    private String name;
    private String skills;
    private int age;
}

File: WEB/exam sample/dotnet/Exam/Data/Entities/SoftwareDeveloper.cs
namespace Data.Entities;

public class SoftwareDeveloper
{
    public int Id { get; set; }
    public string Name { get; set; }
    public int Age { get; set; }
    public string Skills { get; set; }
    
    public List<Project> Projects { get; set; }
}